### PR TITLE
Move show populated to backend

### PR DESCRIPTION
**Pull request by haneslinger**
1. Move "show populated" to backend. zoooooom.
2. add in derived columns.
https://github.com/SEED-platform/seed/issues/4707

### CODE REVIEW

File: seed/static/seed/js/controllers/show_populated_columns_modal_controller.js [hunk 1]
```diff
@@ -17,116 +17,19 @@ angular.module('SEED.controller.show_populated_columns_modal', []).controller('s
...
-        update_profile_with_populated_columns(provided_inventory);
-      } else {
-        const page = 1;
-        const chunk = 5000;
-        fetch(page, chunk).then(update_profile_with_populated_columns);
-      }
-    };
-
     $scope.refresh = () => {
       spinner_utility.show();
```
Comment (perryr16) [inline seed/static/seed/js/controllers/show_populated_columns_modal_controller.js:34]:
```suggestion
      spinner_utility.show();
      $uibModalInstance.close();
```
Might be nice to close the modal once refresh is initiated

**Review by perryr16 — APPROVED** [review]
🏆

**Review by kflemin — CHANGES_REQUESTED** [review]
this seems to work for me on the inventory list page, but I get this error on the inventory detail page and the modal hangs: 
Screenshot 2024-11-04 at 10 24 23 AM

**Comment by haneslinger** [comment]
ah, I forgot about that page. let me at it

**Review by kflemin — APPROVED** [review]

**Pull request merged**